forward-message-mailgun/functions/forward-message-mailgun.js

```diff
@@ -0,0 +1,33 @@
+const mailgun = require("mailgun-js");
+
+exports.handler = function (context, event, callback) {
+  // Configure these at https://www.twilio.com/console/functions/configure
+  const API_KEY = context.MAILGUN_API_KEY;
+  const DOMAIN = context.MG_VERIFIED_DOMAIN;
+  const TO = context.TO_EMAIL_ADDRESS;
+  const FROM = context.FROM_EMAIL_ADDRESS;
+
+  // Instantiate mailgun
+  const mg = mailgun({ apiKey: API_KEY, domain: DOMAIN });
+
+  // Destructure the event
+  const { To, From, Body } = event;
+
+  // Build the message
+  const data = {
+    from: `Excited User <${FROM}>`,
+    to: `${TO}`,
+    subject: `SMS received at Twilio number ${To}`,
+    text: `SMS received from ${From}. \nMessage: ${Body}`,
```

Comment:
Can the messages here match the SendGrid and Sparkpost templates? Also, with my comment above, can we use the `context` variables `TO_EMAIL_ADDRESS` and `FROM_EMAIL_ADDRESS` to keep a distinction between sender and recipients of the SMS and the email.

The `from` email address should just be `FROM_EMAIL_ADDRESS`.
The `to` email address can be `TO_EMAIL_ADDRESS` not `` `${TO}` ``.
The `subject` should be `` `New SMS message from: ${From}` ``.
The `text` should just be `Body`.

If you make those changes, you will no longer need to destructure `To` from the `event`.